packages/xo-web/src/common/intl/messages.js

```diff
@@ -1743,6 +1743,14 @@ const messages = {
     'Are you sure you want to detach {host} from its pool? THIS WILL REMOVE ALL VMs ON ITS LOCAL STORAGE AND REBOOT THE HOST.',
   detachHost: 'Detach',
 
+  // ----- Enable Advanced Live Telemetry -----
+  advancedLiveTelemetry: 'Advanced live telemetry',
```

Comment:
I think we should be consistent with the capitalization. If it's a feature name, maybe we should always capitalize it?